<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="register_PushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repeat Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="passReveal_CheckBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="passReveal_CheckBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailContainAtSign_Lable">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emails should contain @</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerCancel_PushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <zorder>passwordInput_2</zorder>
   <zorder>passwordInput</zorder>
   <zorder>register_PushButton</zorder>
   <zorder>usernameInput</zorder>
   <zorder>emailInput</zorder>
   <zorder>passReveal_CheckBox</zorder>
   <zorder>passReveal_CheckBox_2</zorder>
   <zorder>emailContainAtSign_Lable</zorder>
   <zorder>registerCancel_PushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>usernameInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>passwordInput_2</tabstop>
  <tabstop>emailInput</tabstop>
  <tabstop>passReveal_CheckBox</tabstop>
  <tabstop>passReveal_CheckBox_2</tabstop>
  <tabstop>register_PushButton</tabstop>
  <tabstop>registerCancel_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
